src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/Arcus.Templates.AzureFunctions.Databricks.JobMetrics.csproj

```diff
@@ -24,7 +24,7 @@
     <ContentTargetFolders>content</ContentTargetFolders>
     <EnableDefaultContentItems>false</EnableDefaultContentItems>
     <DockerComposeProjectPath>..\Arcus.Templates.Orchestrator.dcproj</DockerComposeProjectPath>
-    <NoWarn>NU5048;NU5125</NoWarn>
+    <NoWarn>NU5048;NU5125;NU5119</NoWarn>
```

Comment:
Ignore that the `.gitingore` file will not be included in the resulting package.